### PR TITLE
WAVE2 902 Accordian Brick

### PR DESCRIPTION
**Pull request by robbdavis**
WAVE2 902 Accordian Brick

### CODE REVIEW

**Comment by robbdavis** [comment]
Good finds Navinder. These are fixed now.

Note: I also added a fix to make button links open in own window if that option was chosen

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/templates/fieldable-panels-pane--ucgy_gbl_accordion.tpl.php [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+                        $accordion_description = $loaded_entity->field_accordion_description[LANGUAGE_NONE][0]['value'];
+                    }
+
+                    // Accordion button alignment
+                    if (isset($loaded_entity->field_accordion_button_alignment)){
+                        // Define button alignment with lower case value
+                        $button_alignment = strtolower($loaded_entity->field_accordion_button_alignment[LANGUAGE_NONE][0]['value']);
+
+                        // Drop down display value is centre but css uses center
+                        if ($button_alignment == 'centre'){
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/templates/fieldable-panels-pane--ucgy_gbl_accordion.tpl.php]: Is the filter set by centre or center? It appears that both are being used here.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/ucgy_gbl_accordion.module [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    $module_path = drupal_get_path('module', 'ucgy_gbl_accordion');
+    return array(
+        'fieldable_panels_pane__ucgy_gbl_accordion' => array(
+            'variables' => array(),
+            'path' => $module_path . '/templates',
+            'base hook' => 'fieldable_panels_pane',
+            'template' => 'fieldable-panels-pane--ucgy_gbl_accordion'
+        ),
+    );
+}
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/ucgy_gbl_accordion.module]: Missing block comments, should explain what is happening here.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/ucgy_gbl_accordion.module [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @file
+ * Code for the Global Brick: Accordion feature.
+ */
+
+include_once 'ucgy_gbl_accordion.features.inc';
+
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/ucgy_gbl_accordion.module]: Missing block comments, should explain what is happening here.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/ucgy_gbl_accordion.module [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        // Set the core brick class
+        $brick_classes = array('accordion');
+
+        // Merge all the classes together into a happy bundle
+        $vars['classes_array'] = array_merge($vars['classes_array'], $brick_classes);
+    }
+}
+
+
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/ucgy_gbl_accordion.module]: Missing block comments, should explain what is happening here.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/templates/fieldable-panels-pane--ucgy_gbl_accordion.tpl.php (hunk 1 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/templates/fieldable-panels-pane--ucgy_gbl_accordion.tpl.php]: We want the select option to read as 'centre'. The front end devs used the class 'center' in the css. So, this sets the variable to to use the 'center' class if 'centre' was chosen by the user.

Alternatively I could have changed the scss / css to use 'centre', but my scss to css mapping process is suspect at the moment so I just did it this way.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/ucgy_gbl_accordion.module (hunk 4 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/ucgy_gbl_accordion.module]: Added hook_form_alter block comment

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/templates/fieldable-panels-pane--ucgy_gbl_accordion.tpl.php (hunk 1 above)
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_accordion/templates/fieldable-panels-pane--ucgy_gbl_accordion.tpl.php]: General rule: we should avoid using "centre/center" in code, ANYWHERE, and consider it a reserved word due to CSS.

**Comment by robbdavis** [comment]
Hi Geoff. Merge conflicts are sorted.

**Pull request merged**